### PR TITLE
AP_HAL_Chibios: hwdef - ammend CubeYellow and Pixhawk6 board ID's

### PR DESCRIPTION
**Pull request by auturgy**
Follow on from https://github.com/ArduPilot/Bootloader/pull/10

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/Pixhawk6/hwdef-bl.dat [hunk 1]
```diff
@@ -15,7 +15,7 @@ USB_STRING_PRODUCT "Pixhawk6-BL"
 USB_STRING_SERIAL  "%SERIAL%"
 
 # board ID for firmware load
-APJ_BOARD_ID 139
```
Comment (tridge) [inline libraries/AP_HAL_ChibiOS/hwdef/Pixhawk6/hwdef-bl.dat]: pixhawk6 doesn't need a renumber, as "PIXHAWK4PRO" and "Pixhawk6" are the same board

Comment (tridge) [reply libraries/AP_HAL_ChibiOS/hwdef/Pixhawk6/hwdef-bl.dat]: we don't want to use the generic FMUv6 as that will lead to the same issues we had with fmuv5

**Comment by auturgy** [comment]
Pixhawk6 will need a new number.
In syncing the board_types.txt between projects I removed Pixhawk4Pro because that ID was also assigned to JDminiF405.

Regards,

James

**Comment by tridge** [comment]
> Pixhawk6 will need a new number. In syncing the board_types.txt between projects I removed Pixhawk4Pro because that ID was also assigned to JDminiF405. Regards, James

I don't see it, this is the list:
https://github.com/PX4/Bootloader/blob/master/board_types.txt#L54

**Comment by tridge** [comment]
turns out the board ID of 120 was correct, it was just labelled incorrectly in board_types.txt (due to the earlier board rename)
so this PR is not needed

**Pull request closed without merging**